### ISSUE TITLE
Internal error with 0.11.4 on Symfony 4.2.6.

### ISSUE THREAD

**Issue by B-Galati** [phpstan/phpstan-symfony#46]
Hi,

Since I updated to version 0.11.4 on this package I have this error message.

```
#vendor/bin/phpstan analyse -l7 --debug -- src tests config 

PHP Fatal error:  Uncaught PHPStan\ShouldNotHappenException: Internal error. in /opt/project/vendor/phpstan/phpstan-symfony/src/Rules/Symfony/UndefinedArgumentRule.php:52
Stack trace:
#0 /opt/project/vendor/phpstan/phpstan/src/Analyser/Analyser.php(154): PHPStan\Rules\Symfony\UndefinedArgumentRule->processNode(Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\Scope))
#1 /opt/project/vendor/phpstan/phpstan/src/Analyser/NodeScopeResolver.php(1691): PHPStan\Analyser\Analyser->PHPStan\Analyser\{closure}(Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\Scope))
#2 /opt/project/vendor/phpstan/phpstan/src/Analyser/NodeScopeResolver.php(1139): PHPStan\Analyser\NodeScopeResolver->callNodeCallbackWithExpression(Object(Closure), Object(PhpParser\Node\Expr\MethodCall), Object(PHPStan\Analyser\Scope), Object(PHPStan\Analyser\ExpressionContext))
#3 /opt/project/vendor/phpstan/phpstan/src/Analyser/NodeScopeResolver.php(387): PHPStan\Analyser\NodeScopeResolver->processExprNode(Object(PhpParser\Node\Expr\MethodCall) in /opt/project/vendor/phpstan/phpstan-symfony/src/Rules/Symfony/UndefinedArgumentRule.php on line 52
```

My phpstan config:

```
includes:
    - ./vendor/phpstan/phpstan-symfony/extension.neon
    - ./vendor/phpstan/phpstan-webmozart-assert/extension.neon
    - ./vendor/phpstan/phpstan-phpunit/extension.neon
    - ./vendor/phpstan/phpstan-phpunit/rules.neon
    - ./vendor/phpstan/phpstan-doctrine/extension.neon
    - ./vendor/phpstan/phpstan-doctrine/rules.neon
    - ./vendor/jangregor/phpstan-prophecy/src/extension.neon
    - ./vendor/thecodingmachine/phpstan-safe-rule/phpstan-safe-rule.neon
parameters:
    symfony:
        container_xml_path: '%rootDir%/../../../var/cache/dev/srcApp_KernelDevDebugContainer.xml'
    ignoreErrors:
        - message: '#Service ".*" is private\.#'
          path: '%currentWorkingDirectory%/tests'
        - message: '#Cannot .* on Faker\\Generator\|null\.#'
          path: '%currentWorkingDirectory%/tests'
        - message: '#Service "test\..*" is not registered in the container\.#'
          path: '%currentWorkingDirectory%/tests'
```

I am using symfony v4.2.6.

Let me know for anything.

Cheers!

**Comment by lookyman** [comment]
Probably fixed in master (490ff775bd3a6441f6794ae57911d4f9ba347a26), could you test it?

**Comment by B-Galati** [comment]
Indeed!

I am closing this issue.

Thank you!

**Comment by lookyman** [comment]
I've released 0.11.5 for you.
